hippod/report_generator.py

```diff
@@ -468,7 +475,8 @@ def search_last_achievements(app, sha_major, cont_obj):
                 sc = sub_cont['sha-minor']
                 ok, full_sub_cont = hippod.api_shared.read_subcont_obj_by_id(app, sha_major, sc)
                 if not ok:
-                    pass # what if? ---> no raise ApiError possible
+                    pass
+                    # FIXME: what if? ---> no raise ApiError possible
```

Comment:
log error and describe the problem so that the admin know what happend and can provide enough information to the programmer (e.g. you) to fix this.